samples/ingestion/ingestion-client/Connector/Connector.csproj

```diff
@@ -18,6 +18,7 @@
   </PropertyGroup>
   
   <ItemGroup>
+    <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.5.0" />
```

Comment:
The Microsoft.Azure.WebJobs.Extensions.ServiceBus dependency needs to be updated to 5.1.0.